helpers/mathjax.js

```diff
@@ -70,7 +72,8 @@ export class HtmlBlockMathRenderer {
 			});
 
 			await window.MathJax.startup.promise;
-			window.MathJax.typesetShadow(elem.getRootNode(), elem);
+			renderingPromise = renderingPromise.then(() => window.MathJax.typesetShadow(elem.getRootNode(), elem));
```

Comment:
MathJax seems to have made more of its package loading asynchronous, so we now need the shadow handler typesetting to also be asynchronous. This does cause problems if we're typesetting multiple html-blocks on a page, however, because MathJax relies on internal state tracking during rendering, order matters here and we need to make sure only one html-block is ever rendering at a time.

Reply:
Previously the `elem` would have been returned as the promise result of the `render()` call... but now it looks like nothing is returned for the resolved promise. Does this cause any issues for the caller of `render()`?

Reply:
Specifically... https://github.com/BrightspaceUI/core/blob/main/components/html-block/html-block.js#L246

Looks like the code relies on the updated element being returned, so it can subsequently pass it to other renderers.

I'd expect something to be breaking though.

Reply:
Ah, it doesn't break because MathJax does its rendering in place. We still return the element to the outer html-block rendering call, it's just implicitly modified by MathJax's rendering. Because we're passing in a temp element, it stays stable even if MathJax is fully replacing its inner HTML.

Reply:
I worded that poorly. The element being passed around here is already attached to the DOM and MathJax does its work directly on it. There's no need to retrieve an element from MathJax's `render()` function. I believe that was always redundant.

Reply:
Doesn't it return the result of `typesetShadow` and assign it to `elem`?

Reply:
I mean technically it does, but it's redundant. So if anything, changing `typesetShadow` to not return anything makes more sense here. MathJax's actual rendering happens as a side effect.